plasmapy/formulary/tests/test_mathematics.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for the mathematics.py module."""
+
+
+import numpy as np
+import pytest
+
+from plasmapy.formulary import mathematics as mathematics
+
+# TODO: Move the Fermi integral tests over to this file?
+
+
+def test_rot_a_to_b():
```

Comment:
When you add the shape conditioning for `a` and `b`, you'll also need to add a test for it.  I would typically to this like....

```python
@pytest.mark.parametrize(
    "a, b, _raises",
[
        # case 1
        (a, b, ValueError),
        # case 2
        (a, b, ValueError),
        ...
    ],
)
def test_rot_a_to_b_raises(a, b, _raises):
    with pytest.raises(_raises):
        rot_a_to_b(a, b)
```